ApeWorX/ape-solidity
# fix: handle changes from pydantic refactor

### What I did

Adjustments from https://github.com/ApeWorX/ape/pull/509

### How I did it

made work

### How to verify it

can compile solidity contracts again.
make sure to checkout the pydantic refactor branch from ape first!

### Checklist
